packages/core/scripts/gas-rebate/debug/Claims_1.json

```diff
@@ -0,0 +1,426 @@
+{
```

Comment:
This is great for auditability but I really think we should not include these files in the main repo. I could have done the same thing in the weekly yield mining payouts but chose not to do this because each weekly report would then include an additional 1000 line plus file. Over time, this will bloat this repo to an untenable point. 

The Balancer LM repo [here](https://github.com/balancer-labs/bal-mining-scripts) is now 20 gb in size due to all the json output files. I think this is acceptable if the repo is separate but seeing we are putting this in the main protocol repo we should consider keeping the outputs to a minimum.

What do you think? The way I see it is we either want to pull this script into it's own repo where we can dump lots of logs (and do the same for the yield mining script) or remove these outputs for the main protocol repo.

Also keep to hear from @mrice32 & @daywiss on this.

Reply:
I agree that it would be annoying to bloat the repo with this stuff. If this is mainly just a receipt of what happened I think it could either be zipped and/or put in its own repo.

Reply:
@chrismaree what about the actual output.json?

Reply:
100% keep the output.json in the main repo, just like the [liquidity mining payouts](https://github.com/UMAprotocol/protocol/tree/master/packages/core/scripts/liquidity-mining/yusdeth-weekly-payouts).